### ISSUE TITLE
Link in "Complete your bank account setup" message is not pointing to the correct step

### ISSUE THREAD

**Issue by kevinksullivan** [Expensify/App#5088]
(I think this should be made internal due to the #urgency of the issue)

## Action Performed:

1. Create workspace
2. Select Get Started in workspace
3. Go through VBA setup using the [test credentials in (2)](https://stackoverflow.com/c/expensify/questions/342/525#525), and arrive at Validate step 
4. Receive message from concierge that should link to the relevant step, which in this case, should be Requestor Info

## Expected Result:
The link should bring me to the Requestor Info step to review my information on the same tab I was in

## Actual Result:
The link opened another tab, and brought me to the first step of VBA setup, with none of my information saved. The url is also expensify.cash/bank-account/requestor, which should be new.expensify.com/bank-account/requestor.

Related, @JmillsExpensify is seeing another issue with this step [here](https://expensify.slack.com/archives/C020EPP9B9A/p1630704571239700), so it seems like it is broken in several ways.

## Workaround:
The user cannot fix their info with this issue, so it blocks card enablement.

**Notes/Photos/Videos:**
user - kevin@sanfranciscobicycling.us (similar issues for @JmillsExpensify in 

https://user-images.githubusercontent.com/22508290/132076040-fff92b3e-8f10-4c8f-84c7-be3d0493bb41.mov

**Comment by MitchExpensify** [comment]
cc @ctkochan22 @TomatoToaster ([Slack](https://expensify.slack.com/archives/C020EPP9B9A/p1630765306326400?thread_ts=1630717923.321900&cid=C020EPP9B9A))

**Comment by ctkochan22** [comment]
This will be gone if we get rid of double check/pseudo error messages here I realise: https://github.com/Expensify/Expensify/issues/176431#issuecomment-914000119

**Comment by kevinksullivan** [comment]
This is still relevant, explained [here](https://github.com/Expensify/Expensify/issues/176431#issuecomment-914415636).

**Comment by ctkochan22** [comment]
For this one, it says you made it to the validate step?

The video makes it seem that bank account only made it to the VERIFYING state. And the code suggests that message is due to a Requestor error

**Comment by ctkochan22** [comment]
I say this because I wanted to know, what happens if you refresh your page? Or view the bank account form old dot, is it still in validate?

Going to look into the logs and the account.

**Comment by kevinksullivan** [comment]
@ctkochan22 since we chatted 1:1 these questions are covered, right? Just want to make sure you aren't waiting on me here.

**Comment by ctkochan22** [comment]
Oops, no we are good. Hoping to get a PR soon.

**Comment by ctkochan22** [comment]
Dealing with an issue with changes to the link I make on Web-Secure are not persisting into New dot. Asking @marcaaron 1:1 and around for ideas.

**Comment by kevinksullivan** [comment]
Ok so I reproduced this with kevin@expense.business by purposely failing checks, and while I was able to get to the Company Info step the first time around, but it opened a new tab, which is not ideal.

Then the second time around, when I went back into the account, I was unable to get the Company Info step:

https://user-images.githubusercontent.com/22508290/133166924-280bb6cd-e371-4492-ab53-1127add72fe5.mov



So there is still definitely something weird going on here...

**Comment by marcaaron** [comment]
The opening to a new tab thing _should_ be fixed after the changes here https://github.com/Expensify/App/pull/5165/files

**Comment by kevinksullivan** [comment]
That change is [on prod](https://github.com/Expensify/App/pull/5165#issuecomment-917157694) though, right @marcaaron ?

**Comment by marcaaron** [comment]
Yeah it's strange that it's not working. But I see why....

https://github.com/Expensify/Web-Secure/blob/9c9eb0efd6978829a1447a22b7e4eb25a15dbdce/lib/BankAccount/VerificationUtils.php#L209-L210

We are still using the expensify.cash link and need to update it to a new.expensify.com link

**Comment by kevinksullivan** [comment]
Got it. So should we close this issue and create a separate one for that?

**Comment by MelvinBot** [comment]
@ctkochan22 Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by marcaaron** [comment]
>Got it. So should we close this issue and create a separate one for that?

Sorry, I'm not sure - but the solution is there waiting for someone to do it so maybe that would be a good way to move this along.

**Comment by ctkochan22** [comment]
Going to update the link and try again

**Comment by ctkochan22** [comment]
I think we should get the new link pushed and see if it changes. I wonder if the new tab is messing with opening up the bank-account flow with the info in it.

**Comment by kevinksullivan** [comment]
@ctkochan22 let me know what exactly you would like me to test and I'd be happy to knock it out.

**Comment by ctkochan22** [comment]
I just tested it, and it seems like we are good.

**Comment by MelvinBot** [comment]
@ctkochan22 Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!
